pkg/network/tracer/tracer_usm_linux_test.go

```
@@ -802,3 +766,12 @@ func (m requestsMap) String() string {
 
 	return result.String()
 }
+
+func coreSupported() bool {
+	// if we get an error loading the core asset we assume this host doesn't support it
+	err := ddebpf.LoadCOREAsset(&config.New().Config, "http.o", func(bytecode.AssetReader, manager.Options) error {
+		return nil
+	})
+
+	return err == nil
```

Should this use the same strategy as HTTP(S) with `ErrNotSupported`? This seems dangerous and could hide legitimate errors.

---

Also, fwiw there will soon be no exceptions in the CI in terms of BTF availability. https://github.com/DataDog/datadog-agent/pull/15316

---

In fact, you don't need to wait for that PR. The BTF tarball has already been updated.

---

ah, interesting. Let me see if I can remove this altogether then

---

great, all CO-RE tests are passing :smiley: